jupyter/provision-test.ipynb

```diff
@@ -0,0 +1,165 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "provision-test longevity for interactive play with sct cluster"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.environ[\"BUILD_USER_EMAIL\"] = \"sct.tester@scylladb.com\"\n",
+    "os.environ[\"BUILD_ID\"] = \"123\"\n",
+    "os.environ[\"SCT_CLUSTER_BACKEND\"] = \"aws\"\n",
+    "os.environ[\"SCT_CONFIG_FILES\"] = \"test-cases/PR-provision-test.yaml\"\n",
+    "os.environ[\"SCT_SCYLLA_VERSION\"] = \"2022.2.0\"\n",
+    "os.environ[\"SCT_REGION_NAME\"] = \"eu-west-1\"\n",
+    "os.environ[\"SCT_USE_MGMT\"] = \"False\"\n",
+    "os.environ[\"SCT_N_DB_NODES\"] = \"3\"\n",
+    "os.environ[\"SCT_N_LOADERS\"] = \"1\"\n",
+    "os.environ[\"SCT_N_MONITORS_NODES\"] = \"0\"\n",
+    "assert os.environ[\"BUILD_USER_EMAIL\"] != \"sct.tester@scylladb.com\", \"please use your own email so resources are tracked properly\"\n",
+    "\n",
+    "# logging configuration, for jupyter only (sct logs are intact)\n",
+    "import logging\n",
+    "\n",
+    "LOGGER = logging.getLogger(__name__)\n",
+    "LOGGER.setLevel(logging.DEBUG)\n",
+    "consoleHandler = logging.StreamHandler()\n",
+    "consoleHandler.setLevel(logging.DEBUG)\n",
+    "formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')\n",
+    "consoleHandler.setFormatter(formatter)\n",
+    "LOGGER.addHandler(consoleHandler)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "from longevity_test import LongevityTest\n",
+    "import os\n",
+    "\n",
+    "from functional_tests.scylla_operator.libs.auxiliary import sct_abs_path\n",
```

Comment:
Actually it must be this -> `from sdcm import sct_abs_path`.
In general it is better to avoid reimports and in current case it is even more needed here to remove totally unneeded coupling. `Longevity` stuff must not fail if `k8s functional tests` code is broken somehow.